dev/src/reference.ts

```diff
@@ -785,6 +786,8 @@ export class QuerySnapshot {
    *   });
    * });
    */
+  // We deliberately use `any` to allow users to use arbitrary objects as the
+  // callback's context.
```

Comment:
I think you can use `unknown` here since we don't care what the type is, we just pass it through.

Reply:
Yes, that works. Not sure how `.call()` is defined, but the linter doesn't complain.